PermissionsUtilsUI/src/main/res/layout/activity_app_diagnosis.xml

```diff
@@ -0,0 +1,242 @@
+            android:layout_width="@dimen/dk_ic_xxbig"
+            android:layout_height="@dimen/dk_ic_xxbig"
+            android:layout_margin="@dimen/dk_margin_half"
+            android:layout_marginBottom="@dimen/dk_margin"
+            android:padding="@dimen/dk_margin_quarter"
+            android:src="@drawable/ic_checked"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/text_view_summary_title"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/dk_margin_half"
+            android:text="@string/dk_perm_utils_diag_app_ok"
+            android:textSize="16dp"
+            android:textStyle="bold"
+            app:layout_constraintStart_toEndOf="@+id/image_view_summary_icon"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/text_view_summary_description"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/dk_margin_half"
+            android:text="@string/dk_perm_utils_diag_app_ok_text"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/image_view_summary_icon"
+            app:layout_constraintTop_toBottomOf="@+id/text_view_summary_title" />
+
+        <TextView
+            android:id="@+id/summary_view_separator"
+            style="@style/Separator.Horizontal"
+            android:layout_marginTop="@dimen/dk_margin_half"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/text_view_summary_description" />
+
+        <com.drivequant.drivekit.permissionsutils.commons.views.DiagnosisItemView
+            android:id="@+id/diag_item_location"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/dk_margin_half"
+            app:diagnosisLink="@string/dk_perm_utils_app_diag_location_link"
+            app:diagnosisSubtitle="@string/dk_perm_utils_app_diag_learn_more"
+            app:diagnosisTextKO="@string/dk_perm_utils_app_diag_location_ko_android"
+            app:diagnosisTextOK="@string/dk_perm_utils_app_diag_location_ok"
+            app:diagnosisTitle="@string/dk_perm_utils_app_diag_location_title"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/summary_view_separator" />
+
+        <com.drivequant.drivekit.permissionsutils.commons.views.DiagnosisItemView
+            android:id="@+id/diag_item_notification"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:diagnosisLink="@string/dk_perm_utils_app_diag_notification_link"
```

Comment:
@mohamed-garci I'm not sure the strings can be override if they are set here

Reply:
@Steven-DriveQuant override how ?. (it's not clear)

Reply:
@mohamed-garci For example, if a client wants to override the `dk_perm_utils_app_diag_notification_link` key, will it work ?

Reply:
@Steven-DriveQuant i guess yes.